.env.j2

```diff
@@ -0,0 +1,23 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Trello token and API key for utilising Trello REST API
+{% block content %}
```

Comment:
We shouldn't need these `block content` labels, that's set up in our HTML templates as a way to work with the layout template but isn't necessary (though nor harmful) here